README.md

```diff
@@ -1 +1,3 @@
 # Apple_Clone
+
+This project is an attempt to reproduce an old version of the [Apple's Website](https://web.archive.org/web/20140301004610/http://www.apple.com/) using HTML and CSS
```

Comment:
#### `HINT`

Add a descriptive readme. Details like assignment link (from Odin), live link (from [githack](https://raw.githack.com/)), screenshot and what was learned during the project can make the project more descriptive. :+1: